test/utils/mocks/body.html

```diff
@@ -26,6 +26,10 @@
     <p><a href="https://milo.adobe.com/img/favicon.svg">https://milo.adobe.com/img/favicon.svg</a></p>
     <p><a href="https://www.adobe.com">https://milo.adobe.com/img/favicon.svg</a></p>
     <p><a class="bad-url" href="https://www.adobe.com">img/favicon.svg</a></p>
+    <!-- Video -->
+    <p><a href="https://main--blog--adobecom.hlx.page/media_17927691d22fe4e1bd058e94762a224fdc57ebb7b.mp4">
+      https://main--blog--adobecom.hlx.page/media_17927691d22fe4e1bd058e94762a224fdc57ebb7b.mp4
+    </a></p>
```

Comment:
You add the anchor to mocks, but I don't see a corresponding test for `decorateVideo`?

Reply:
I added a test.